app/javascript/packages/document-capture/components/in-person-call-to-action.tsx

```diff
@@ -0,0 +1,46 @@
+import { useContext } from 'react';
+import { Button, Tag } from '@18f/identity-components';
+import { useInstanceId } from '@18f/identity-react-hooks';
+import { t } from '@18f/identity-i18n';
+import { formatHTML } from '@18f/identity-react-i18n';
+import AnalyticsContext from '../context/analytics';
+
+function InPersonCallToAction() {
+  const instanceId = useInstanceId();
+  const { trackEvent } = useContext(AnalyticsContext);
+
+  return (
+    <section
+      aria-labelledby={`in-person-cta-heading-${instanceId}`}
+      aria-describedby={`in-person-cta-tag-${instanceId}`}
+    >
+      <hr className="margin-y-5" />
+      <Tag id={`in-person-cta-tag-${instanceId}`}>
+        {t('in_person_proofing.body.cta.new_feature')}
+      </Tag>
+      <h2 id={`in-person-cta-heading-${instanceId}`} className="margin-y-2">
+        {t('in_person_proofing.headings.cta')}
+      </h2>
+      <p>
+        <strong>
+          {formatHTML(t('in_person_proofing.body.cta.prompt'), {
+            wbr: 'wbr',
+          })}
+        </strong>{' '}
+        {t('in_person_proofing.body.cta.prompt_detail')}
+      </p>
+      <Button
+        isBig
+        isOutline
+        isWide
+        href="#location"
+        className="margin-top-3 margin-bottom-1"
+        onClick={() => trackEvent('IdV: verify in person troubleshooting option clicked')}
```

Comment:
I might choose to `useCallback` on `trackEvent` and memoize the output of `formatHTML` to avoid some re-renders.

Reply:
In this case, I think it might be overkill, and would add overhead both in terms of [performance cost](https://kentcdodds.com/blog/usememo-and-usecallback) and readability. This component is rendered as a leaf of the render tree, so I wouldn't expect additional renders to have much of an impact. I think it can make more sense in components which have many descendants, such that a render could cause a cascade of child renders.

Reply:
It would also be a problem to have a lot of leaves w/ this kind of approach, but I don't think we have enough complexity on the page for a performance problem here.